### PR TITLE
Separated Referral, Consultation, Public Comment into their own tabs

### PR DESCRIPTION
**Pull request by mmumby**
- Separated Referral, Consultation, Public Comment into their own tabs
- All functionality remains the same, This move will allow us to start tracking each stage with current sprints progress work
Screen Shot 2020-11-06 at 4 24 41 PM
Screen Shot 2020-11-06 at 4 25 01 PM
Screen Shot 2020-11-06 at 4 25 11 PM

- Updated the logic of the NOWStatusBanner to include other types of icons since they are all driving off the same logic
-Banner
-Badges
- Timeline (Didn't add this yet, but thinking it could live in this component as well due to the similar logic/colouring)
- Also updated styling, added color  references to the Alert Banner (Still hidden for now)
Screen Shot 2020-11-06 at 4 24 16 PM

- Added Badges to the tab headers (Hidden for now until conditions are implemented)
Screen Shot 2020-11-06 at 5 18 48 PM

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/NOWStatusIndicator.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/* eslint-disable */
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/NOWStatusIndicator.js:1]: Should we remove this and address any linting issues?

File: services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js [hunk 2]
```diff
@@ -861,35 +870,94 @@ export class NoticeOfWorkApplication extends Component {
             </Tabs.TabPane>
 
             <Tabs.TabPane
-              tab="Referral/Consultation"
-              key="referral-consultation"
+              tab={
+                <span>
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js:874]: Can we pull this out into a function, as this is being used in the other tabs and they should follow the same format.
```
function renderTabTitle = (title) =>
    (
                <>
                  <NOWStatusIndicator type="badge" />
                  {title}
                </>
    );
```
for example.

**Review by exzizt — CHANGES_REQUESTED** [review]
Looks really good! Just some minor comments. 👍

File: services/core-web/src/components/noticeOfWork/NOWStatusIndicator.js (hunk 1 above)
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/NOWStatusIndicator.js:1]: Yes, was planning to remove once this component is live/has all the logic - This is just a starting point to have this component available for Ryan and myself.

File: services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js (hunk 2 above)
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js:874]: Yes, I can make this change.

Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js]: Updated!

**Review by exzizt — APPROVED** [review]
✔️

**Pull request merged**